### ISSUE TITLE
Add installer strings to transifex

### ISSUE THREAD

**Issue by smarquespt** [luspi/photoqt-dev#36]
Is it possible to export installer strings to transifex?

**Comment by luspi** [comment]
I actually have been playing around with translating the installer before. So far, I haven't quite got it to work - the problem is not extracting the strings and uploading them on Transifex, but loading the right strings from some translation file... I know it is possible (somehow). Let me try again, and I'll let you know how I get on with it.

**Comment by smarquespt** [comment]
Don´t know how but smplayer does that. https://www.assembla.com/code/smplayer/subversion/nodes/6724/smplayer/trunk/setup/translations

**Comment by luspi** [comment]
Thanks for the link, I had a partial success using their solution already... I'll keep you posted!

**Comment by luspi** [comment]
PhotoQt is moving away from GitHub to GitLab ([more info](http://photoqt.org/news/1434375887)). This issue is duplicated here: https://gitlab.com/luspi/photoqt-legacy/issues

**Comment by luspi** [comment]
Transifex is now also used for Windows Installer Strings.
Closing this. If you have further comments, please use https://gitlab.com/luspi/photoqt-legacy/issues/16
